src/test/operators/aggregate_test.cpp

```diff
@@ -184,19 +184,20 @@ class AggregateSortedTest : public BaseTest {
     const auto chunk_count = test_table->chunk_count();
     for (ChunkID chunk_id = ChunkID{0}; chunk_id < chunk_count; ++chunk_id) {
       const auto chunk = test_table->get_chunk(chunk_id);
-      const auto ordered_by = chunk->ordered_by();
-      if (ordered_by) {
-        const auto ordered_segment = chunk->get_segment(ordered_by->first);
+      const auto ordered_by_vector = chunk->ordered_by();
+      if (ordered_by_vector) {
```

Comment:
Putting it here because there is not join test, yet.

Since the  reason for the multiple-column sort was to keep the sorting information of joins, we definitely need a test for that.

Reply:
Done in [Commit](https://github.com/hyrise/hyrise/pull/2000/commits/da599a4ea64983d981d6fe7c73af1704b9c80c31)